frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/activity-card/activity-card.tsx

```diff
@@ -52,9 +48,7 @@ export const getOCSSubscription = (subscriptions: FirehoseResult): SubscriptionK
 };
 
 const ocsEventNamespaceKindFilter = (event: EventKind): boolean =>
-  getNamespace(event) === CEPH_STORAGE_NAMESPACE ||
-  _.get(event, 'involvedObject.kind') ===
-    (PersistentVolumeClaimModel.kind || PersistentVolumeModel.kind);
+  getNamespace(event) === CEPH_STORAGE_NAMESPACE;
```

Comment:
wouldn't this pop up all the noobaa alerts too on persistent storage ?

Reply:
That  happening now as well.
I just corrected the logic which makes no sense.

Reply:
Then why you removed the PV PVC events?

Reply:
The logic makes absolute sense, The PVC can be created in any namespace & we will not be able to see the events

Reply:
```
 _.get(event, 'involvedObject.kind') ===
    (PersistentVolumeClaimModel.kind || PersistentVolumeModel.kind);
```
This is incorrect here as this will always check for PVC and not PV.
If  PV/PVC required then we can do :
```
 event?.involvedObject?.kind === PersistentVolumeClaimModel.kind ||
  event?.involvedObject?.kind === PersistentVolumeModel.kind;
```
When operator `||` finds a truthy value, it returns that.
Hence, 
```
 _.get(event, 'involvedObject.kind') ===
    (PersistentVolumeClaimModel.kind || PersistentVolumeModel.kind);
```
is equivalent to 
```
 _.get(event, 'involvedObject.kind') === PersistentVolumeClaimModel.kind;
```

Reply:
Also I think we should show here ceph based PVCs and PVs only aligning with inventory card.

Reply:
For this [comment](https://github.com/openshift/console/pull/5363#discussion_r424175310)
I will raise a followup PR.

Reply:
Thanks for the explanation, use this for this PR or just don't touch it as this is not part of the bug you are trying to fix.
```
 event?.involvedObject?.kind === PersistentVolumeClaimModel.kind
```